internal/backends/mcu/fonts.rs

```diff
@@ -7,14 +7,66 @@ use core::cell::RefCell;
 #[cfg(all(not(feature = "std"), feature = "unsafe_single_core"))]
 use i_slint_core::thread_local_ as thread_local;
 
-use super::renderer::ScaleFactor;
-use i_slint_core::graphics::{BitmapFont, BitmapGlyph, BitmapGlyphs, FontRequest, Size};
+use crate::{LogicalLength, LogicalSize, PhysicalLength, PhysicalSize, ScaleFactor};
+use euclid::num::Zero;
+use i_slint_core::{
+    graphics::{BitmapFont, BitmapGlyph, BitmapGlyphs, FontRequest},
+    slice::Slice,
+};
 
 thread_local! {
     static FONTS: RefCell<Vec<&'static BitmapFont>> = RefCell::default()
 }
 
-pub const DEFAULT_FONT_SIZE: f32 = 12.;
+#[derive(Copy, Clone)]
+pub struct Glyph(&'static BitmapGlyph);
+
+impl Glyph {
```

Comment:
Ideally BitmapGlyph would be using PhysicalRect or something like that.

Reply:
Yes. That requires the compiler to emit that and the lengths types to be in i_slint_core. That's weird because of the PhysicalLength using i16 as scalar type.